pyproject.toml

```diff
@@ -29,6 +29,7 @@ dependencies = [
     "semver>=2.13.0,<4.0",
     "six>=1.16.0,<2.0.0",
     "tqdm>=4.59.0,<5.0.0",
+    "werkzeug>=2.3.7,<3.0.0"
```

Comment:
😞 but ok

Reply:
yeah.... it was the lightest-weight (I think this only pulls in `MarkupSafe`) and most user-friendly thing I found while looking through some options